leaflet.shpfile.js

```diff
@@ -1,8 +1,11 @@
 L.Shapefile =L.GeoJSON.extend({
-    initialize: function (file, options) {
+    initialize: function (file, options, importUrl) {
         if(typeof cw !== 'undefined'){
+            if (typeof importUrl === 'undefined') {
```

Comment:
The "style" of the file seemed to prefer `typeof` checking over the shorter methods, but I can go either way.